testnode-scripts/config.ts

```diff
@@ -250,6 +250,39 @@ function writeConfigs(argv: any) {
     fs.writeFileSync(path.join(consts.configpath, "poster_config.json"), JSON.stringify(posterConfig))
 }
 
+function writeL2ChainConfig(argv: any) {
+    const l2ChainConfig = {
+		"chainId": 412346,
+		"homesteadBlock": 0,
+		"daoForkSupport": true,
+		"eip150Block": 0,
+		"eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+		"eip155Block": 0,
+		"eip158Block": 0,
+		"byzantiumBlock": 0,
+		"constantinopleBlock": 0,
+		"petersburgBlock": 0,
+		"istanbulBlock": 0,
+		"muirGlacierBlock": 0,
+		"berlinBlock": 0,
+		"londonBlock": 0,
+		"clique": {
+			"period": 0,
+			"epoch": 0
+		},
+		"arbitrum": {
+			"EnableArbOS": true,
+			"AllowDebugPrecompiles": true,
+			"DataAvailabilityCommittee": false,
+			"InitialArbOSVersion": 11,
+			"InitialChainOwner": "0x0000000000000000000000000000000000000000",
```

Comment:
Now that we have support for variable chain configs, could we set the initial chain owner here instead of relying on the debug precompile to become owner? If possible I'd even like to make the node binary fail to startup if the chain config has AllowDebugPrecompiles set to true. This was really a hacky workaround for the fact that the chain config wasn't dynamically settable, which we've now fixed.

Reply:
I'll make a separate issue for this